frontend/src/components/UIEditor/DatabaseTree.vue

```diff
@@ -140,12 +141,7 @@ const contextMenuOptions = computed(() => {
     });
     return options;
   } else if (treeNode?.type === "table") {
-    const table = editorStore.findTable(
-      treeNode.tableId,
-      treeNode.label,
-      treeNode.databaseId
-    );
-    const isDropped = editorStore.droppedTableList.includes(table as Table);
+    const isDropped = editorStore.droppedTableList.includes(treeNode.table);
```

Comment:
I'm not if this will be dangerous when we use `deepClone(table)` somewhere, changing the references.
To prevent that happening we may find the table by id. e.g, `editorStore.droppedTableList.find(table => table.id === treeNode.table.id)`;
If we are sure that the object reference won't change, this is okey.

Reply:
That's okay